docs/website/root/manual/getting-started/run-signer-node.md

```diff
@@ -113,6 +113,30 @@ Note that this guide works on a Linux machine only.
 
 :::
 
+:::info
+
+Compare the version of your Cardano node with the minimum supported versions listed in the [`cardano-min-versions.json`](https://github.com/input-output-hk/mithril/blob/main/cardano-min-versions.json) to verify its compatibility with the Mithril signer.
+
+First, check the version of your Cardano node by running the following command:
+
+```bash
+cardano-node --version
+```
+
+Then, refer to the minimum supported versions listed in the the [`cardano-min-versions.json`](https://github.com/input-output-hk/mithril/blob/main/cardano-min-versions.json) file.
+
+You can also fetch the minimum supported version for your network using the command below:
+```bash
+wget -q -O - https://raw.githubusercontent.com/input-output-hk/mithril/main/cardano-min-versions.json | jq -r '.network.**YOUR_CARDANO_NETWORK**'
+```
+
+Here is an example for `preprod`:
+```bash
+wget -q -O - https://raw.githubusercontent.com/input-output-hk/mithril/main/cardano-min-versions.json | jq -r '.network.preprod'
+```
```

Comment:
This should be updated with new JSON format

Reply:
Done ✅